frontend/lib/norent/letter-builder/error-pages.tsx

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import Page from "../../ui/page";
+import { Link } from "react-router-dom";
+import { NorentRoutes } from "../routes";
+import { CustomerSupportLink } from "../../ui/customer-support-link";
+
+type ErrorPageProps = {
+  title: string;
+  children: React.ReactNode;
+};
+
+type CtaLinkProps = {
+  to: string;
+  children: React.ReactNode;
+};
+
+const CtaLink: React.FC<CtaLinkProps> = (props) => (
+  <p className="has-text-centered">
+    <Link className="button is-primary is-large jf-is-extra-wide" to={props.to}>
+      {props.children}
+    </Link>
+  </p>
+);
```

Comment:
Hey @toolness, have you considered using the `<CenteredPrimaryButtonLink>` button component here? Looks almost identical to what you've implemented here.

Reply:
ohhh good point, the funny thing is that I actually pulled this code from some other existing code on a different NoRent page, so I filed #1326 to address it during cleanup!